src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2105,5 +2080,16 @@ private IInstanceReferenceOperation CreateCollectionValuePlaceholderOperation(Bo
             bool isImplicit = placeholder.WasCompilerGenerated;
             return new InstanceReferenceOperation(referenceKind, _semanticModel, syntax, type, constantValue, isImplicit);
         }
+
+        private IVariableDeclarationGroupOperation CreateBoundUsingLocalDeclarationOperation(BoundUsingLocalDeclarations boundUsingLocalDeclarations)
+        {
+            bool isAsync = boundUsingLocalDeclarations.AwaitOpt is object;
+            VariableDeclarationKind declarationKid = isAsync ? VariableDeclarationKind.AsynchronousUsing : VariableDeclarationKind.Using;
+            SyntaxNode declarationSyntax = ((LocalDeclarationStatementSyntax)boundUsingLocalDeclarations.Syntax).Declaration;
```

Comment:
>(LocalDeclarationStatementSyntax) [](start = 44, length = 33)

It would be more robust to not assume that the cast is going to succeed. For example, a compiler generated node doesn't have to be associated on a local declaration statement syntax.